docs/ignoring-files-folders-code.md

```diff
@@ -88,6 +88,12 @@ Unsupported patterns are silently removed from the pattern list (this is done so
 
 For a description of `.gitignore` syntax, see [.gitignore documentation](https://git-scm.com/docs/gitignore).
 
+:::caution
+[Semgrepignore is being revised](/semgrepignore-v2-reference) to
+support multiple `.semgrepignore` files and match the Gitignore
```

Comment:
```suggestion
support multiple `.semgrepignore` files and match the `.gitignore`
```
I went through both the GitHub and git documentation and they consistently refer to Gitignore as `.gitignore`.

Citations:
- https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository#_ignoring
- https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files

Reply:
See my earlier long comment about "Gitignore" vs. "`gitignore`".

Reply:
It's the "Gitignore" specification. "`.gitignore`" is a file name.